docs/drain-aquifer.rst

```diff
@@ -39,3 +42,5 @@ Options
     Remove all aquifers on the current z-level and above.
 ``-z``, ``--cur-zlevel``
     Remove all aquifers on the current z-level.
+``-f``, ``--filter [light/heavy]``
+    Remove all light or heavy aquifers.
```

Comment:
Since there are only two options here, it might be cleaner to call the options `-l, --light` and `-h, --heavy` for draining only light or heavy aquifers, respectively.